### ISSUE TITLE
508/a11y: Services page - multiple H1s

### ISSUE THREAD

**Issue by iamjolly** [gsa-oes/office-of-evaluation-sciences#338]
On the [Services page](https://federalist-9dc074d2-9bb0-4720-8e37-036899ae1463.app.cloud.gov/preview/gsa-oes/office-of-evaluation-sciences/2021-website-updates-2.0/services/), there are two `h1` elements. 

Each page should have only one main heading (`h1`). 

From the [18F Accessibility Guide on Headings](https://accessibility.18f.gov/headings/):
> When using heading elements, reserve the `<h1>` element for the page title. On the home page, this is usually the title of the site and on other pages, this may be the page title. Use the `<h1>` element once per page—some assistive technology may be unable to read multiple `<h1>`s on a single page correctly. Other heading levels may be used more than once following document outline order.

Recommendation: 
Change the `h1` around the text "We partner with agencies across government to:" to be `p class="billboard-message` to be more consistent with other pages on the site and reduce possible (albeit minor) confusion about the main point of the page.

**Comment by alyssagalik** [comment]
updated to billboard message
